packages/amplify-sdk/src/amplify-sdk.js

```diff
@@ -405,6 +405,18 @@ export default class AmplifySDK {
 			}
 		};
 
+		this.entitlement = {
+			/**
+			 * Retrieves entitlement information for a specific entitlement metric.
+			 * @param {Object} account - The account object.
+			 * @param {String} metric - The entitlement metric name.
+			 * @returns {Promise<Object>}
+			 */
+			find: (account, metric) => this.request(`/api/v1/entitlement/${metric}`, account, {
+				errorMsg: 'Failed to get entitlement info'
+			})
```

Comment:
Super minor; missing semicolon.

Reply:
Not quite. `find` is a prop being assigned a function and you don't put semicolons inside an object definition.

Reply:
My mistake; thanks.